spec/support/test_grpc_web_app.rb

```diff
@@ -9,7 +9,8 @@
 # Used to build a Rack app hosting the HelloService for integration testing.
 module TestGRPCWebApp
   def self.build(service_class = TestHelloService)
-    GRPCWeb.handle(service_class)
```

Comment:
@noaelad Was there a reason you switched this test utility to use the singleton RackApp instance instead of initializing a fresh one on every test? To avoid the chance of test pollution I think it's better to use a fresh instance, but wanted to check with you before changing it back.

Reply:
I wanted us to have some test coverage for the `GRPCWeb::RackApp` class =]
We can add coverage in a different way though, maybe a unit test instead